etc/profile-a-l/koi.profile

```diff
@@ -0,0 +1,53 @@
+# Firejail profile for koi
+# Description: Theme scheduling for the KDE Plasma Desktop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include koi.local
+# Persistent global definitions
+include globals.local
+
+# Restriction below breaks program on Arch.
+#include disable-common.inc
+
+include disable-devel.inc
```

Comment:
```suggestion
include disable-common.inc
include disable-devel.inc
```

disable-common.inc is a rather important include.

I'd suggest including it and commenting lines in it until you find which ones
are causing problems.

Then for the relevant lines add `ignore <entry>` before the include.